preproc/LED_charge_summary.ipynb

```diff
@@ -0,0 +1,1043 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.hist(df_filtered[df_filtered['channel_id'] == 34].efine_corrected, bins = 50)\n",
+    "plt.xlabel('Efine corrected')\n",
+    "plt.ylabel('Entries')\n",
+    "plt.title('Efine corrected for channel 34')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "bfa15cc2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def filtered_mean(df):\n",
+    "    '''In each channel we have got two well differentiated \n",
+    "    distributions. With this function, we filter the gaussian\n",
+    "    which is due to LED contribution only.'''\n",
+    "    \n",
+    "    mean_vector = 0\n",
+    "    if df.mean() > 30:\n",
+    "        mean_vector = df[df > 30].mean() #we select the gaussian on the right\n",
+    "    elif df.mean() < 30:\n",
+    "        mean_vector = df.mean() # we select all data\n",
+    "        \n",
+    "    return mean_vector\n",
+    "        "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "68d8e66d",
+   "metadata": {},
+   "source": [
+    "We obtain the charge sum and charge mean for each channel and we create a matrix with the correct position:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "7dc2d027",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "A channel efine corrected mean is:  78.06400937422241\n"
+     ]
+    }
+   ],
+   "source": [
+    "channel_sums  = df_filtered.groupby('sensor_id')['efine_corrected'].sum().reset_index()\n",
+    "charge_sum    = np.array(channel_sums['efine_corrected']).reshape(8,8)\n",
+    "\n",
+    "channel_means = df_filtered.groupby('sensor_id')['efine_corrected'].apply(filtered_mean).reset_index()\n",
+    "charge_mean   = np.array(channel_means['efine_corrected']).reshape(8,8)\n",
+    "\n",
+    "print('A channel efine corrected mean is: ', channel_means.efine_corrected.mean())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f2674ddf",
+   "metadata": {},
+   "source": [
+    "**Sum of the charge per channel**"
```

Comment:
Is this the sum of the charge of a specific sensor over many events? Why is this number relevant?

Reply:
I have talked with Vicente, as we did this together and it is not relevant, so I have just remove it.

Reply:
Maybe I'm not looking at the right plot, but it seems to me that you haven't removed it.

Reply:
Sorry, I thought you were only talking about the print, not about the plot, and I didn't realize it was the same. I have removed it too.